packages/mux-uploader/src/mux-uploader-progress.ts

```diff
@@ -139,20 +183,22 @@ class MuxUploaderProgressElement extends globalThis.HTMLElement {
       this.setAttribute('type', ProgressTypes.BAR);
     }
 
-    if (currentType === ProgressTypes.RADIAL) {
-      if (this.svgCircle) {
-        // strokeDasharray is the size of dashes used to draw the circle with the size of gaps in between.
-        // If the dash number is the same as the gap number, no gap is visible: a full circle.
-        // strokeDashoffset defines where along our circle the dashes (in our case, a dash as long as the
-        // circumference of our circle) begins. The larger the offset, the farther into the circle you're
-        // starting the "dash". In the beginning, offset is the same as the circumference. Meaning, the visible
-        // dash starts at the end so we don't see the full circle. Instead we see a gap the size of the circle.
-        // When the percentage is 100%, offset is 0 meaning the dash starts at the beginning so we can see the circle. (TD).
-        this.svgCircle.style.strokeDasharray = `${this.getCircumference()} ${this.getCircumference()}`;
-        this.svgCircle.style.strokeDashoffset = `${this.getCircumference()}`;
-      }
+    if (currentType === ProgressTypes.RADIAL && this.svgCircle) {
+      // strokeDasharray is the size of dashes used to draw the circle with the size of gaps in between.
+      // If the dash number is the same as the gap number, no gap is visible: a full circle.
+      // strokeDashoffset defines where along our circle the dashes (in our case, a dash as long as the
+      // circumference of our circle) begins. The larger the offset, the farther into the circle you're
+      // starting the "dash". In the beginning, offset is the same as the circumference. Meaning, the visible
+      // dash starts at the end so we don't see the full circle. Instead we see a gap the size of the circle.
+      // When the percentage is 100%, offset is 0 meaning the dash starts at the beginning so we can see the circle. (TD).
+      this.svgCircle.style.strokeDasharray = `${this.getCircumference()} ${this.getCircumference()}`;
+      this.svgCircle.style.strokeDashoffset = `${this.getCircumference()}`;
     }
   }
+
+  formatProgress(percent: number): string {
```

Comment:
suggestion(non-blocking): If this isn't intended to be overridden, consider making it a separate function instead of an instance method.